code/game/objects/machinery/door_display/door_display.dm

```diff
@@ -13,99 +13,102 @@
 	desc = "A remote control for a door."
 	anchored = TRUE
 	density = FALSE
+	interaction_flags = INTERACT_MACHINE_TGUI
+	///Does this type allow for flashers?
+	var/has_flash = FALSE
```

Comment:
Why add this if its unused

Reply:
it is used

Reply:
7 references and only one I see it set is here